scripts/collect_env.py

```diff
@@ -0,0 +1,281 @@
+import subprocess
+import platform
+import pkg_resources
+
+
+def command_exists(command):
+    """
+    Check if a command exists in the system
+
+    Args:
+        command (str): The command to check
+
+    Returns:
+        bool: True if command exists, False otherwise
+    """
+    try:
+        subprocess.run([command], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
Can we look for the command in `PATH` instead of running the command? This seems like it would execute the command.

Reply:
Alternatively, you could make this `brew_command_exists` and have it do `brew help` since I only see it checking for `brew` in the script.